Sources/_RegexParser/Regex/Parse/Source.swift

```diff
@@ -9,10 +9,12 @@
 //
 //===----------------------------------------------------------------------===//
 
-/// The source given to a parser. This can be bytes in memory, a file on disk,
-/// something streamed over a network connection, etc.
+// For now, we use String as the source while prototyping...
+
+/// The source of text being given to a parser.
```

Comment:
QUERY: Is this where the regex comes from, or where text the regex matches comes from?

Reply:
`Source` represents the text of a regular expression when parsing and compiling the pattern. It isn't used while applying a regex to an input text.